domain/user.js

```diff
@@ -25,6 +27,30 @@ class User {
   async viewNotifications() {
     return await this.userRepository.fetchAllNotifications(this.id.getId());
   }
+
+  async sendNonUserEmail(userEmail, inviteMessage) {
+    const transporter = nodemailer.createTransport({
+      host: 'smtp.gmail.com',
+      port: 587,
+      secure: false, // true for 465, false for other ports
+      auth: {
+        user: process.env.COVICARE_EMAIL, // generated ethereal user
```

Comment:
Same as before, inside this function we should call a single function that handles the sending of emails